src/streamlink/plugins/rtpplay.py

```diff
@@ -25,7 +26,8 @@ def _get_streams(self):
             log.error("Could not find _m3u8_re")
             return
 
-        hls_url = m.group("url")
+        hls_url = eval("\"\".join((" + m.group(1) + "))")
```

Comment:
don't use `eval`

Reply:
Get the array and its content without the `.join()` call and parse it as JSON. Then do the array to string conversion and decoding afterwards in the plugin logic.

Reply:
This does obviously only work as long as the site doesn't add more "code obfuscation" which would require a JS runtime, like for example dynamic string concatenation or function calls, etc. Some sites do this so their sources can't be statically parsed, but this seems simple enough.